configs/caas/cromwell.conf.ctmpl

```diff
@@ -222,8 +222,8 @@ call-caching {
 
   # Whether to invalidate a cache result forever if we cannot reuse them. Disable this if you expect some cache copies
   # to fail for external reasons which should not invalidate the cache (e.g. auth differences between users):
-  # (default: true)
-  invalidate-bad-cache-results = false
+  # (default: false)
```

Comment:
Revert this. You didn't change the default.

Reply:
I changed this because https://cromwell.readthedocs.io/en/stable/Configuring/#call-caching says that the default is `false`.. Is that incorrect?

Reply:
Woops, I misread it. Will change.